README.md

```diff
@@ -117,6 +121,17 @@ To check they've been applied correctly, either:
 * `supabase db diff` to run against the local database
 * `supabase migration list` on both dev and target databases can be compared
 
+#### Seed the local database
+The local database is generated based on the code in `seed.mts`. This uses Snaplet to generate a large amount of data.
+To generate te SQL queries needed to populate database, run
+```shell
+npx tsx seed.mts > supabase/seed.sql
+```
+To rebuild the database from the migration files and `seed.sql`, run
+```shell
+npm run dev:reset_supabase
+```
+
```

Comment:
Should these local setup steps include the createStorage and create-admin-user tools, or have I missed where they are run?

Reply:
`dev:reset_supabase` script actually resets the database using the seed, and calls `createStorage` and `createAdminAndCallerUsers`, since we want do go through those steps whenever the database is reset.

Reply:
Oh yeah, sorry I didn't read package.json properly